playbooks/setup.yml

```diff
@@ -5,6 +5,8 @@
   remote_user: '{{ remote_user | default("root") }}'
   gather_facts: '{{ gather_facts | default("yes") }}'
   roles:
+    - { role: clean, when: skip_clean|bool == false }
+    - { role: packages, when: skip_packages|bool == false }
     - { role: prereqs, when: skip_prereqs|bool == false }
-    - { role: minishift, when: setup_minishift|bool == true }
-    - role: pipeline
\ No newline at end of file
+    - { role: artifacts, when: skip_artifacts|bool == false }
```

Comment:
artifacts maybe not the right name for the role

Reply:
I don't like much this name too but I don't have any other suggestion now. Can you recommend a proper name for this role?
The intention of this role I mentioned above.